drivers/video/ov5640.c

```diff
@@ -743,12 +752,8 @@ static int ov5640_set_ctrl_brightness(const struct device *dev, int value)
 {
 	const struct ov5640_config *cfg = dev->config;
 
-	if (!IN_RANGE(value, -UINT8_MAX, UINT8_MAX)) {
-		return -EINVAL;
-	}
-
 	struct ov5640_reg brightness_params[] = {{SDE_CTRL8_REG, value >= 0 ? 0x01 : 0x09},
-						 {SDE_CTRL7_REG, abs(value) & 0xff}};
+						 {SDE_CTRL7_REG, (abs(value) << 4) & 0xf0}};
```

Comment:
```suggestion
						 {SDE_CTRL7_REG, (abs(value) << BIT(2)) & GENMASK(15,8)}};
```

Reply:
I believe you mean (abs(value) << BIT(4)) & GENMASK(7,4)}}.

Could you explain the benefit of using these macros here ?

IMO, GENMASK() will help if we need to generate something dynamically that is only known at runtime like GENMASK(x, y) or in case it is difficult to do the calculation, e.g. GENMASK(63, 51). Here, the mask is very clear and simple,  we just put `0xf0` why have to pass by this macro ?